### PR TITLE
show logging buffered

### LINKED ISSUE #113: show logging buffered

### PR DESCRIPTION
**Pull request by LondonWesley**

### CODE REVIEW

**Comment by LondonWesley** [comment]
I was confused on what I would check for with the file

File: asa_parser/src/asa_parse.py [hunk 1]
```diff
@@ -205,40 +205,40 @@ def show_cpu_detailed(self):
         return json.dumps({'text': self.get_show_section('cpu detailed')})
 
     def ipsec_stats(self):
-        """Parser for show ipsec stats"""
```
Comment (claesmk) [inline asa_parser/src/asa_parse.py]: It looks like you are changing the indentation on ipsec_stats() here - why is that?

**Review by claesmk — CHANGES_REQUESTED** [review]

File: asa_parser/src/asa_parse.py (hunk 1 above)
Comment (LondonWesley) [reply asa_parser/src/asa_parse.py]: Oh sorry. I had a section commented out because it was giving errors.

**Comment by claesmk** [comment]
Looks like it is still not working - see the failed check - you should be able to leave the IPSec_stats() section unmodified and it should work OK - not sure why there would be any errors.

**Comment by claesmk** [comment]
Fixed spacing issues in https://github.com/NCATComp410/comp410_fall_2020/pull/119

**Pull request closed without merging**